bats/tests/k8s/up-downgrade-k8s.bats

```diff
@@ -70,10 +71,24 @@ verify_images() {
     verify_images
 }
 
+# Remove all the kubectl clients from the .kuberlr directory.
+# Then run `kubectl`, and it should pull in the `kubectl` for
+# the current k8s version in that directory.
+
+verify_kuberlr_for_version() {
+    k8sVersion=$1
+    rm -f "${HOME}"/.kuberlr/"${OS}"-"${ARCH_FOR_KUBERLR}"/kubectl*
```

Comment:
Having so many quotes looks noisy:

```suggestion
    rm -f "${HOME}/.kuberlr/${OS}-${ARCH_FOR_KUBERLR}/kubectl"*
```

Reply:
Also maybe use a variable for the path, as you use it again 3 lines later. Makes sure both lines are always in sync.

```suggestion
    local kuberlr_cache="${HOME}"/.kuberlr/"${OS}"-"${ARCH_FOR_KUBERLR}"
    rm -f "${kuberlr_cache}/kubectl"*
```